README.md

```diff
@@ -86,6 +86,11 @@ proxies, and collect telemetry data from proxies and services. There
 are plugins for interfacing with various cloud platforms, policy
 management services, and monitoring services.
 
+    - [operator](operator/). This directory contains code for
+[_istio operator_](https://istio.io/docs/reference/commands/operator/). You can read operator installation instructions
+[here](https://istio.io/docs/setup/install/standalone-operator/). If you have a bug backport for istio operator 1.4
```

Comment:
The workflow stuff is a little suspect and will need to be removed once 1.4 is deprecated (which would be once this repo is released). This PR is backportable, however, so I'll submit a backport request, and then I recommend a followup PR to remove the "if you have a bug report.." part.

Reply:
> a followup PR to remove the "if you have a bug report.." part

@sdake Sounds good to me. I'll work on this now.